src/YMenuButton.cc

```diff
@@ -133,19 +134,85 @@ YMenuButton::findMenuItem( int wantedIndex, YItemConstIterator begin, YItemConst
     return 0;
 }
 
-
-void
-YMenuButton::resolveShortcutConflicts()
+static void resolveShortcutsConflictFlat(YItemIterator begin, YItemIterator end)
 {
-    // TO DO
-    // TO DO
-    // TO DO
+    bool used[ sizeof( char ) << 8 ];
+    for ( unsigned i=0; i < sizeof( char ) << 8; i++ )
```

Comment:
byte cannot be larger then 8bits, even ShortCut manager have it same way, so it probably works.